src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -139,18 +139,13 @@ export const DeploymentAppSelector = ({
 }
 
 export const ChartProjectSelector = ({
-    isDeployChartView,
     selectedProject,
     handleProjectSelection,
     projects,
     invalidProject,
 }: ChartProjectSelectorType): JSX.Element => {
-    return !isDeployChartView ? (
```

Comment:
Don't we need to render the project read-only filed in the values tab?

Reply:
No, this is fine, chart project selector will only project selector and instead of read only project update modal is introduced.